{{cookiecutter.repostory_name}}/pyproject.toml

```
@@ -1,20 +1,64 @@
 [project]
 name = "{{cookiecutter.repostory_name}}"
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

```suggestion
requires-python = "==3.11"
```
for library it would make sense to support a range, but for application it makes package resolution needlessly complicated